### ISSUE TITLE
Tab label text does not draw in correct Windows theme colour in high contrast mode(s)

### ISSUE THREAD

**Issue by blairmcg** [dolphinsmalltalk/Dolphin#1077]
When Windows 10 is in High Contrast Dark mode, the text labels of tabs should be white, but are still drawn in black so invisible.

![image](https://user-images.githubusercontent.com/15128107/102689917-c7d1a700-41f9-11eb-9ba9-93719707cc95.png)

It would be nice if the source code theme adapted to the background colour as well, but that is more of a feature enhancement.
